Packs/ShiftManagement-AssignToNextShift/Playbooks/playbook-Assign_Active_Incidents_to_Next_Shift.yml

```
@@ -169,9 +169,13 @@ tasks:
     skipunavailable: false
     task:
       brand: ""
-      description: "Randomly assigns the incidents to users on call (requires shift\
-        \ management) and users on call.  \n\nhttps://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Shift-Management#idf554fd0f-f93b-40cd-9111-1393bf25ac6e\n\
-        \nIncident Ids should be passed in as a comma separated list."
+      description: "Randomly assigns the incidents to users on call (requires shift management) and users on call.
```

```suggestion
      description: "Randomly assigns the incidents to users on call (requires shift management) and users on call.
```
Is the second "users on call" supposed to be there?  If so, maybe it should be "users on call (with no shift management)."